gadget/validate_test.go

```diff
@@ -252,28 +430,13 @@ volumes:
 	// only content for the first volume
 	makeSizedFile(c, filepath.Join(s.dir, "first.img"), 1, nil)
 
-	err := gadget.Validate(s.dir, nil, nil)
+	ginfo, err := gadget.ReadInfo(s.dir, nil, nil)
+	c.Assert(err, IsNil)
+	err = gadget.ValidateContent(ginfo, s.dir)
 	c.Assert(err, ErrorMatches, `invalid layout of volume "second": cannot lay out structure #0 \("second-foo"\): content "second.img": stat .*/second.img: no such file or directory`)
 }
 
-func (s *validateGadgetTestSuite) TestValidateBorkedMeta(c *C) {
```

Comment:
this test seems to have disappeared ?

Reply:
I see equivalent tests are already in gadget_test.go so this is fine that this test goes away